src/main/java/uk/gov/ida/verifyserviceprovider/configuration/HubEnvironment.java

```diff
@@ -33,16 +52,35 @@ public static HubEnvironment fromString(String name) {
             ));
     }
 
-    HubEnvironment(URI ssoLocation, URI metadataUri) {
+    HubEnvironment(URI ssoLocation, URI metadataUri, String expectedEntityId, String trustStoreName) {
         this.ssoLocation = ssoLocation;
         this.metadataUri = metadataUri;
+        this.expectedEntityId = expectedEntityId;
+        this.trustStoreName = trustStoreName;
     }
 
     public URI getSsoLocation() {
         return this.ssoLocation;
     }
 
-    public URI getMetadataUri() {
-        return this.metadataUri;
+    public HubMetadataConfiguration getMetadataConfiguration() {
+        return new HubMetadataConfiguration(
+                this.metadataUri, null, null,
```

Comment:
Could maybe be explicit about these `null` values but they are defaulted in the base class to sensible values so didn't want to repeat myself

Reply:
As discussed: it might be worth adding an overload so that it's clearer that default values will be used, rather than null arguments magically being superseded by default values.

Reply:
I agree with Alan, it is not clear without further digging that the base class default is used here